earn/src/components/lend/BorrowingWidget.tsx

```diff
@@ -181,8 +273,12 @@ export default function BorrowingWidget(props: BorrowingWidgetProps) {
                   <div className='flex flex-col'>
                     {borrowers &&
                       borrowers.map((account) => {
-                        const hasNoCollateral = account.assets.token0Raw === 0 && account.assets.token1Raw === 0;
+                        const hasNoCollateral =
+                          account.assets.token0Raw === 0 &&
+                          account.assets.token1Raw === 0 &&
+                          (account.uniswapPositions?.length || 0) === 0;
```

Comment:
In theory it would show up (so I can fix that), however with the current set of contract calls we make, it shouldn't be possible to end up with an empty Uniswap position still stored in slot0. Users would have to manually make their own calls in order to get into that situation.